include/SFML/OpenGL.hpp

```diff
@@ -33,7 +33,7 @@
 
 
 ////////////////////////////////////////////////////////////
-/// This file just includes the OpenGL (GL and GLU) headers,
+/// This file just includes the OpenGL headers,
```

Comment:
All GLU functionality can be reproduced with pure GL and user code. Think of them like... "macros" that call some very very ancient GL code. It is widely established (even since before modern GL became a thing) that GLU should be avoided whenever possible, which is almost always.

It is not guaranteed that GLU will even be available on all platforms. Some (look at the mobile platforms) might have even thrown it out already.